lib/helpers.js

```diff
@@ -30,7 +30,8 @@ unslugify = function (name) {
  * unrecognized.
  */
 
-var yesPrimary = ['yes', 'yeah', 'y', 'ya', 'yea', 'yup', 'yep', 'true', 'sure', 'gladly', 'absolutely', 'totally', 'totes', 'got it', 'ok', 'k', 'kk', 'umkay', 'sounds good', 'indeed', 'no problem', 'not a problem', 'okay', 'yess', 'yesss', 'yas', 'good', 'so far, so good', 'so far so good', 'of course']
+const neither = ['not sure'] // This is interpreted as 'yes' without a special case
```

Comment:
It seems like" not sure" is interpreted as neither neither yes or no. So, I think i understand the code but the comment here contradicts my understanding. Can you clarify the comment.

Reply:
Ah i think i see what your saying now. It would have been interpretted as yes previously, without this new special case. I think going forward this would be confusing as without context of the change, it says to me that this is sometimes "not sure" is interpretted as 'yes'